src/data/multiset/basic.lean

```diff
@@ -1001,6 +1001,30 @@ lemma prod_map_mul [comm_monoid γ] {m : multiset α} {f g : α → γ} :
   prod (m.map $ λa, f a * g a) = prod (m.map f) * prod (m.map g) :=
 multiset.induction_on m (by simp) (assume a m ih, by simp [ih]; cc)
 
+@[simp, to_additive]
+lemma prod_map_inv' [comm_group γ] {m : multiset α} {f : α → γ} :
```

Comment:
Can you use this to golf finset.prod_inv_distrib?

Reply:
Yes indeed. That makes it look like there are ways to golf my own new lemmas.